packages/presentation/lib/screen/home_data.dart

```diff
@@ -0,0 +1,16 @@
+class PalindromeData {
+  bool isPalindrome;
+  String inputPalindrome;
+
+  PalindromeData(this.isPalindrome, this.inputPalindrome);
+
+  factory PalindromeData.init() => PalindromeData(false, '');
+
+  void call({
+    required bool? isPalindrome,
+    required String? inputPalindrome,
+  }) {
+    if (isPalindrome != null) this.isPalindrome = isPalindrome;
+    if (inputPalindrome != null) this.inputPalindrome = inputPalindrome;
+  }
```

Comment:
what is this method needed for?

Reply:
updating properties in the instance PalindromeData classes.